cpp/examples/mcapdump.cpp

```diff
@@ -96,9 +96,9 @@ std::string ToString(const mcap::ChunkIndex& chunkIndex) {
     "[ChunkIndex] start_time={}, end_time={}, chunk_start_offset={}, chunk_length={}, "
```

Comment:
@jhurliman do we need to update this or can we perhaps remove mcapdump now that we have a c++ conformance test runner?